src/app/components/file-uploader/file-uploader.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container" app-drag-and-drop>
+    <input type="file" #fileDropRef id="fileDropRef" (change)="fileBrowseHandler($event.target)"
+        [multiple]="acceptMultiple" [accept]="acceptedTypes.join(',')" />
+    <h3>Drag and drop a file here</h3>
+    <h3>Or</h3>
+    <label for="fileDropRef">Browse for a file here</label>
+</div>
+<div class="files-list">
+    <div class="single-file" *ngFor="let file of files; let i = index">
+        <div class="file-info">
+            <img alt="image-preview" class="file-icon" [src]=file?.preview >
+            <h4 class="file-name">
+                {{ file?.name }}
+            </h4>
+            <h4 class="file-size">
+                {{ file?.readableSize }}
+            </h4>
+
+            <div class="delete" (click)="deleteFile(i)">
```

Comment:
Take a look at how the remove collaborators is styled, I think it would be nice if the remove file button would be the same